docs/pages/identity-governance/access-requests/resource-requests.mdx

```diff
@@ -261,450 +252,15 @@ Getting updated certificates...
-to any Kubernetes resources, including the entire Kubernetes cluster.
-
-The following role allows users to request access to Kubernetes namespaces.
-Requests for Kubernetes resources other than `pods` will not be allowed.
-
-```yaml
-kind: role
-metadata:
-  name: requester-kube-access
-version: v8
-spec:
-  allow:
-    request:
-      search_as_roles:
-        - kube-access
-      kubernetes_resources:
-        - kind: pods
-          api_group: ''
-```
-
-The following role allows users to request access only to Kubernetes deployments and/or pods.
-
-```yaml
-kind: role
-metadata:
-  name: requester-kube-access
-version: v8
-spec:
-  allow:
-    request:
-      search_as_roles:
-        - kube-access
-      kubernetes_resources:
-        - kind: deployments
-          api_group: apps
-        - kind: pods
-          api_group: ''
-```
-
-The following role allows users to request access to any specific Kubernetes resources.
-
-```yaml
-kind: role
-metadata:
-  name: requester-kube-access
-version: v8
-spec:
-  allow:
-    request:
-      search_as_roles:
-        - kube-access
-      kubernetes_resources:
-        - kind: '*'
-          api_group: '*'
-```
-
-The `request.kubernetes_resources` field only restricts what
-`kind` / `api_group` of Kubernetes resource requests are allowed.
-To control Kubernetes access to these resources see
-[Kubernetes Resources](../../enroll-resources/kubernetes-access/controls.mdx)
-section for more details.
-
-#### `db`
-
-You can restrict access to searching `db` resources by assigning values to the
-`db_labels` field in the `spec.allow` or `spec.deny` fields.
-
-The following role allows access to databases with the `env:dev` or
-`env:staging` labels:
-
-```yaml
-kind: role
-version: v5
-metadata:
-  name: developer
-spec:
-  allow:
-    db_labels:
-      env: ["dev", "staging"]
-
-    # Database account names this role can connect as.
-    db_users: ["viewer", "editor"]
-    db_names: ["*"]
-```
-
-#### `app`
-
-You can restrict access to searching `app` resources by assigning values to the
-`app_labels` field in the `spec.allow` or `spec.deny` fields.
-
-The following role allows access to all applications except for those in
-`env:prod`:
-
-```yaml
-kind: role
-version: v5
-metadata:
-  name: dev
-spec:
-  allow:
-    app_labels:
-      "*": "*"
-  deny:
-    app_labels:
-      env: "prod"
-```
-
-#### `windows_desktop`
-
-You can restrict access to searching `windows_desktop` resources by assigning
-values to the `windows_desktop_labels` field in the `spec.allow` or `spec.deny`
-fields.
-
-The following role allows access to all Windows desktops with the
-`env:dev` or `env:staging` labels.
-
-```yaml
-kind: role
-version: v4
-metadata:
-  name: developer
-spec:
-  allow:
-    windows_desktop_labels:
-      env: ["dev", "staging"]
-
-    windows_desktop_logins: ["{{internal.windows_logins}}"]
-```
-
-### Request access to Kubernetes resources
```

Comment:
I think it's find if the tsh related content lives under clients section but I do think that it's needed under main Identity Governance section as well, even if it's a link to the respective tsh guide. Otherwise I will be wondering how to actually use and test what I've just configured.

Reply:
The how-to guide in Identity Governance includes basic `tsh` commands for testing out the resource request flow, but it does make sense to include a link to the user guide. I'll add that.